client/src/__tests__/errorBoundary.test.tsx

```diff
@@ -0,0 +1,36 @@
+import "jest-dom/extend-expect";
+import "react-testing-library/cleanup-after-each";
+
+import React from "react";
+import { render } from "react-testing-library";
+import ErrorBoundary from "../ErrorBoundary";
+
+jest.mock("../Header", () => ({
+  HEIGHT: 50
+}));
+jest.mock("../basePath", () => "");
```

Comment:
In terms of application flow for me it makes most sense to have these `jest.mock` calls even before the `import` statements since they tinker with the stuff that is then being imported.

(In reality this makes no difference since jest internally moves the mocks up and uses `require` instead of `import`)